<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reportes Emes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);

</string>
  </property>
  <widget class="QLineEdit" name="advanceFile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>620</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="suppliersFile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>620</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toFile">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>620</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="advancePath">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>60</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}
</string>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 150, 214);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:500;&quot;&gt;Reporte 260&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 150, 214);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:500;&quot;&gt;Proveedores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 150, 214);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:500;&quot;&gt;Guardar en&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:20px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}

</string>
   </property>
   <property name="text">
    <string>Cargar archivos</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>520</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:20px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}</string>
   </property>
   <property name="text">
    <string>Comparar descuentos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useButton">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>520</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:20px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}</string>
   </property>
   <property name="text">
    <string>Incluir aprovechamiento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suppliersPath">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}</string>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toPath">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:15px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>611</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,255);
border-image: url(:/cct/logo_emes.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseSuppliersButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>520</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 150, 214);
	color: rgb(255, 255, 255);
	border-style: solid;
	border-width:2px;
	border-radius:20px;
}
QPushButton:hover {
	background-color: rgb(141, 206, 233);
}</string>
   </property>
   <property name="text">
    <string>Seleccionar proveedores</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>advanceFile</zorder>
  <zorder>suppliersFile</zorder>
  <zorder>toFile</zorder>
  <zorder>advancePath</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>uploadButton</zorder>
  <zorder>runButton</zorder>
  <zorder>useButton</zorder>
  <zorder>suppliersPath</zorder>
  <zorder>toPath</zorder>
  <zorder>chooseSuppliersButton</zorder>
 </widget>
 <resources>
  <include location="logo_emes.qrc"/>
 </resources>
 <connections/>
</ui>
